backends-velox/src/main/scala/org/apache/gluten/config/VeloxConfig.scala

```diff
@@ -112,6 +112,13 @@ object VeloxConfig {
       .timeConf(TimeUnit.MILLISECONDS)
       .createWithDefault(TimeUnit.MINUTES.toMillis(60))
 
+  val COLUMNAR_VELOX_MEMORY_POOL_CAPACITY_TRANSFER_ACROSS_TASKS =
+    buildConf("spark.gluten.sql.columnar.backend.velox.memoryPoolCapacityTransferAcrossTasks")
```

Comment:
The config key name is too long. I know it aligns with the current name pattern. Maybe, better to simplify all such config names to shorter ones, e.g., `spark.gluten.velox.xxx`, in the future`. cc @yikf

Reply:
agreed.